### PR TITLE
updating ios & android install links

### PR DESCRIPTION
**Pull request by Lndsay**
Updating iOS & Android Maps SDK installation links to align with install updates to mapbox.com

### CODE REVIEW

File: platform/android/README.md [hunk 1]
```diff
@@ -98,4 +98,4 @@ More information about building and distributing this project in [DISTRIBUTE.md]
 #### Symbolicating native crashes
 
 When hitting native crashes you can use ndk-stack to symbolicate crashes. 
-More information in [this](https://github.com/mapbox/mapbox-gl-native/wiki/Getting-line-numbers-from-an-Android-crash-with-ndk-stack) guide.
\ No newline at end of file
+More information in [this](https://github.com/mapbox/mapbox-gl-native/wiki/Getting-line-numbers-from-an-Android-crash-with-ndk-stack) guide.
```
Comment (Lndsay) [inline platform/android/README.md]: @zugaldia I didn't touch anything here so i'm not sure why/what changes being tracked are. Would love 👀  to make sure we're good to go.

File: platform/ios/INSTALL.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Integrating custom builds of the Mapbox iOS SDK into your application
 
-This document explains how to build a development version of Mapbox iOS SDK for use in your own Cocoa Touch application. To use a production-ready version of the SDK, see the [Mapbox iOS SDK homepage](https://mapbox.com/ios-sdk).
+This document explains how to build a development version of Mapbox iOS SDK for use in your own Cocoa Touch application.
```
Comment (boundsj) [inline platform/ios/INSTALL.md]: I know we expect people to reach this via a particular flow, but it is also very possible that someone may end up on this page directly. Would it be bad to keep the (redundant with the root INSTALL.md) notice about the pre-built options available at https://www.mapbox.com/install?

Comment (Lndsay) [reply platform/ios/INSTALL.md]: ohh very good point. I don't think it'd be bad to have a little redundancy to account for a scenario like that.  Will update.

File: platform/ios/README.md [hunk 3]
```diff
@@ -4,9 +4,10 @@
...
 
-This repository is for day-to-day development of the SDK. Building the SDK yourself requires [a number of dependencies and steps](../../INSTALL.md) that are unnecessary for developing production applications. For production applications, please consider installing an official, prebuilt release instead; see the [Mapbox iOS SDK website](https://www.mapbox.com/ios-sdk/) for installation instructions.
+This repository is for day-to-day development of the SDK. Building the SDK yourself requires [a number of dependencies and steps](../../INSTALL.md) that are unnecessary for developing production applications. For production applications, please consider installing an official, production-ready version instead.
 
-* [Integrating the Mapbox iOS SDK into your application](INSTALL.md)
+* [Integrate a production-ready version of the Mapbox iOS SDK into your application](https://www.mapbox.com/install/ios/).
+* [Integrate custom builds of the Mapbox iOS SDK into your application](INSTALL.md)
```
Comment (boundsj) [inline platform/ios/README.md]: I think ideally we would always encourage the pre-built versions to put "into your application". Maybe we could word this in a way that would frame it more as "developing and contributing" to the project? 

"Build, develop, and contribute to the Mapbox iOS SDK" ?

I know that conflicts with the description for DEVELOPING.md just below, but if you read it, `DEVELOPING.md` is almost completely about tips and tricks for modern C++ development. I think INSTALL.md gives iOS developers more useful information about how to get set up to contribute to the project.

File: platform/android/README.md (hunk 1 above)
Comment (zugaldia) [reply platform/android/README.md]: @Lndsay not sure either - everything looks fine and link is still working, let's 🚢 .

**Comment by Lndsay** [comment]
Thanks everyone for your 👀 , and extra thanks to @friedbunny for the protips and flagging the typo in my email address! 🙇‍♀️

**Pull request merged**